### DISCUSSION TITLE
Export the namespace/interface Detox from index.d.ts

### DISCUSSION THREAD

**Discussion by reinaldonetof** [wix/Detox#3571] [Q&A]
Hello, guys! is it possible to export the namespace/interface Detox from `index.d.ts`? With that, it would be possible to import the Detox interface to a ts file and type the tests correctly.

**Comment by d4vidi** [comment]
I think work for that is under way in the context of Detox 20. @noomorph `Y/n?`

**Comment by reinaldonetof** [comment]
@d4vidi The namespace Detox can be used and the TS is working properly, but the eslint is complaining about the namespace Detox is not defined.

Also, do you know if is it possible to [fix Detox typings to not clash with Jest's](https://github.com/solkaz/ts-detox-example/issues/2), instead of importing expect in all tests files?
